tsdb/chunks/head_chunks.go

```diff
@@ -767,7 +757,11 @@ func (cdm *ChunkDiskMapper) DeleteCorrupted(originalErr error) error {
 
 // Size returns the size of the chunk files.
 func (cdm *ChunkDiskMapper) Size() int64 {
-	return cdm.size.Load() + cdm.curFileSize()
+	size, err := fileutil.DirSize(cdm.dir.Name())
```

Comment:
Let's return this error so that it doesn't go unnoticed (we return the error for WAL size calculation)